src/analyze/analyze.c

```diff
@@ -2169,7 +2170,7 @@ static int do_security(int argc, char *argv[], void *userdata) {
         (void) pager_open(arg_pager_flags);
 
         if (arg_security_policy) {
-                r = json_parse_file(/*f=*/ NULL, arg_security_policy, /*flags=*/ 0, &policy, &line, &column);
+                r = json_parse_file(/*f=*/ NULL, arg_security_policy, arg_pager_flags, &policy, &line, &column);
```

Comment:
JsonParseFlags is not the same as PagerFlags

Reply:
@maanyagoenka this is what's causing the build failure